.github/workflows/verify.yml

```diff
@@ -1,24 +1,32 @@
-name: Java CI
+name: Java CI with Gradle
 
 on:
   push:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+  workflow_dispatch:
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Check out
+      uses: actions/checkout@v2
+
     - name: Set up JDK 11
       uses: actions/setup-java@v2
       with:
         java-version: '11'
         distribution: 'adopt'
-    - name: Build with Maven
-      run: mvn -B package --file pom.xml
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v2
+
+    - name: Setup Gradle
+      uses: gradle/gradle-build-action@v2
+
+    - name: Setup Gradlew
+      run: chmod +x ./gradlew
```

Comment:
to trzeba inaczej, lokalnie w gicie robisz git update-index --chmod=+x gradlew i commit&push do repo, ten chmod tutaj nie będzie potrzebny

Reply:
done